certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json

```diff
@@ -1631,8 +1682,8 @@
                                                         "count": {
                                                             "type": "integer",
                                                             "format": "int32",
-                                                            "description": "Count",
-                                                            "title": "Count"
+                                                            "description": "Count of connected domains sharing the IP country_code",

```

Comment:
@RamboV should it be `country code` here or do you want it to be `country_code`?